app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,46 @@
+class RentalsController < ApplicationController
+
+  def checkout
+    customer_id = params[:customer_id]
+    rental = Rental.new(rental_params)
+    rental.start_date = Date.today
+    rental.end_date = Date.today + 7
+
+    if rental.save
+      customer = Customer.find_by(id: customer_id)
+      customer.movies_checked_out_count += 1
```

Comment:
All of this work around assigning dates, decreasing `movies_checked_out_count`, etc. should be encapsulated in  one testable model method.